packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationMembers.tsx

```diff
@@ -71,16 +80,29 @@ export const OrganizationMembers = withCardStateProvider(() => {
           <Tabs>
             <TabsList sx={t => ({ gap: t.space.$4 })}>
               {canReadMemberships && (
-                <Tab localizationKey={localizationKeys('organizationProfile.membersPage.start.headerTitle__members')} />
+                <Tab localizationKey={localizationKeys('organizationProfile.membersPage.start.headerTitle__members')}>
+                  <NotificationCountBadge
+                    notificationCount={membershipsCount}
+                    colorScheme='neutral'
+                  />
+                </Tab>
               )}
               {canManageMemberships && (
                 <Tab
                   localizationKey={localizationKeys('organizationProfile.membersPage.start.headerTitle__invitations')}
-                />
+                >
+                  <NotificationCountBadge
+                    notificationCount={invitationsCount}
+                    colorScheme='neutral'
+                  />
+                </Tab>
               )}
               {canManageMemberships && isDomainsEnabled && (
                 <Tab localizationKey={localizationKeys('organizationProfile.membersPage.start.headerTitle__requests')}>
-                  <NotificationCountBadge notificationCount={membershipRequests?.count || 0} />
+                  <NotificationCountBadge
+                    notificationCount={membershipRequestsCount}
+                    colorScheme='neutral'
+                  />
```

Comment:
Previously we had an animation happening on mount, do we still want that ? Or is this removed ? Just checking here i'm good either way.

Reply:
Changing some logic to keep the way the animation is trigger, thank you!